cpukit/libmisc/cpuuse/cpuusagetop.c

```diff
@@ -273,10 +273,10 @@ task_usage(Thread_Control* thread, void* arg)
               CPU_usage_Less_than(&usage, &data->usage[j]))
             continue;
         case RTEMS_TOP_SORT_REAL_PRI:
-          if (thread->real_priority > data->tasks[j]->real_priority)
+          if (thread->Priority_node.real_priority > data->tasks[j]->Priority_node.real_priority)
             continue;
         case RTEMS_TOP_SORT_CURRENT_PRI:
-          if (thread->current_priority > data->tasks[j]->current_priority)
+          if (thread->Priority_node.current_priority > data->tasks[j]->Priority_node.current_priority)
```

Comment:
Check for line lengths exceeding 80 characters, and wrap them if they do. This should be relatively easy to detect by skimming your commit files.

Reply:
https://devel.rtems.org/wiki/Developer/Coding/80_characters_per_line